### DISCUSSION TITLE
Template options

### DISCUSSION THREAD

**Discussion by rabbitism** [AvaloniaUI/Avalonia.Templates#254] [Q&A] [answered]
I can see there are project template options  (like choose from ReactiveUI and CommunityToolkit) and it is available in VS. but I cannot see them in Rider. Is there anything wrong?

**Comment by Mrxx99** [comment, marked as answer]
Yes, there is something wrong with Rider. It does not display dotnet template options at all. (I guess the options they have on other templates are custom made by Rider). But it should work with next public release: https://youtrack.jetbrains.com/issue/RIDER-16759/Support-parameters-in-custom-project-templates

**Comment by maxkatz6** [comment]
Rider didn't support them until recently. Should be available in their preview versions.

**Comment by rabbitism** [comment]
Thank you all for the info!
